### ISSUE TITLE
[question] [enhancement] Add possibility to choose used theme

### ISSUE THREAD

**Issue by benedyktdryl** [filipovskii/gulp-doxx#6]
I want to use ie. _cayman_\* template. Can we add option which allow to use custom theme?

---

Mentioned theme is valid doxx theme, producing documentation:

https://www.npmjs.com/package/doxx
http://fgribreau.github.io/doxx/docs/compiler.js.html

**Comment by flpvsk** [comment]
Doesn't [theme](https://github.com/filipovskii/gulp-doxx#template) option work?

UPD: I meant `template` option, of course. But It seems like it's not the same as theme. Is it?

**Comment by benedyktdryl** [comment]
Yes - it's not same thing. Doxx authors strongly recommended to switch to themes, despite templates are not marked as deprecated.

**Comment by flpvsk** [comment]
I see. Will take a look at it during the week. Thanks!
